.github/dependabot.yml

```
@@ -0,0 +1,24 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  # Maintain dependencies for go-modules
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: weekly
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: weekly
+
+  # Maintain dependencies for dockerfile
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

```suggestion
      interval: weekly
```

Looks good, does this need quotes?

---

it is yaml config, either way is fine :)

---

in the example, you can see it is quoted
https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#scheduleday

---

my query was more around the inconsistency. we have 2 unquoted and 1 not

---

I got you, let me fix it. :)

---

all good now.